token/services/db/sql/identity.go

```diff
@@ -79,17 +212,39 @@ func (w *WalletPathStorageIterator) HasNext() bool {
 
 func (w *WalletPathStorageIterator) Next() (driver.IdentityConfiguration, error) {
 	var c driver.IdentityConfiguration
+	c.Type = w.configurationType
 	err := w.rows.Scan(&c.ID, &c.URL)
 	return c, err
 }
 
 func (db *IdentityDB) GetSchema() string {
 	return fmt.Sprintf(`
-		-- Identities
+		-- IdentityConfigurations
 		CREATE TABLE IF NOT EXISTS %s (
 			identity_id TEXT NOT NULL PRIMARY KEY,
+            type TEXT NOT NULL,  
```

Comment:
Maybe an index on the type might speed up a bit the queries.

Reply:
will do.